README.md

```diff
@@ -241,6 +241,32 @@ This SDK exposes the following classes:
 - `TransactionsWidget`
 - `TrendsWidget`
 
+### Troubleshooting
```

Comment:
We could have other sections here in the future. For example, we could add an "OAuth redirects not working" section (specially useful for the mobile SDKs.)